static/scss/top-app-bar.scss

```
@@ -1,14 +1,20 @@
 // sass-lint:disable mixins-before-declarations
 
 .header-holder {
-  padding: 20px 1px;
+  padding: 20px 0 5px;
```

1. The `ul` for the menu has a bottom margin. If we set `ul {margin: 0}` somewhere in here, then the menu will be vertically centered 
2. It would be better to have consistent top and bottom padding set on this container element. 20px is also pretty large. The designs might call for something like that, but I think it's safe to just set this to 10px for now and review later

```suggestion
  padding: 10px 0;
```